libvcx/src/libindy/utils/wallet.rs

```diff
@@ -49,6 +60,43 @@ pub fn reset_wallet_handle() {
     agency_wallet::reset_wallet_handle();
 }
 
+pub fn create_wallet_from_config(config: &str) -> VcxResult<()> {
+    let config: WalletConfig = serde_json::from_str(config)
+        .map_err(|err| VcxError::from_msg(VcxErrorKind::InvalidJson, format!("Cannot deserialize WalletConfig {:?}, err: {:?}", config, err)))?;
+
+    let wh = create_and_open_as_main_wallet(
+        &config.wallet_name,
+        &config.wallet_key,
+        &config.wallet_key_derivation,
+        config.wallet_type.as_ref().map(String::as_str),
+        config.storage_config.as_ref().map(String::as_str),
+        config.storage_credentials.as_ref().map(String::as_str),
+    )?;
+    set_wallet_handle(wh);
```

Comment:
Isn't wallet handle already set in `create_and_open_as_main_wallet `?

Reply:
Good catch